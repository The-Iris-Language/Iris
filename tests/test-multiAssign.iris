.. tests multiple assignment using '.'
class Main () {
  public:
    int main() {
      Animal.bark();
      Dog.bark();
      Dog.age = 0;
      Dog.age = Cat.age = Fish.age = Gecko.age;
      return 0;
    }
}