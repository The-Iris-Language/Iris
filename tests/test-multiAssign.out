class Main of Object () {
public:
int main() {
Animal.bark();
Dog.bark();
Dog.age = 0;
Dog.age = Cat.age = Fish.age = Gecko.age;
return 0;
}
}